tailcfg/tailcfg.go

```diff
@@ -79,7 +79,8 @@ type CapabilityVersion int
 //   - 40: 2022-08-22: added Node.KeySignature, PeersChangedPatch.KeySignature
 //   - 41: 2022-08-30: uses 100.100.100.100 for route-less ExtraRecords if global nameservers is set
 //   - 42: 2022-09-06: NextDNS DoH support; see https://github.com/tailscale/tailscale/pull/5556
-const CurrentCapabilityVersion CapabilityVersion = 42
+//   - 43: 2022-09-14: adds Debug.EnableSilentDisco
+const CurrentCapabilityVersion CapabilityVersion = 43
```

Comment:
I don't think we want to bump this quite yet as it doesn't work yet.

Control can send this bit blindly to nodes regardless of whether they actually support it and old nodes will ignore it, so I don't think we want to advertise this until it works. Meanwhile you can make control send it to just your node during testing. (or really, use the env var)